src/m365/aad/commands/app/app-get.ts

```diff
@@ -119,7 +120,9 @@ class AadAppGetCommand extends GraphCommand {
       throw `No Azure AD application registration with ${applicationIdentifier} found`;
     }
 
-    throw `Multiple Azure AD application registration with name ${name} found. Please disambiguate (app object IDs): ${res.value.map(a => a.id).join(', ')}`;
+    const resultAsKeyValuePair = formatting.convertArrayToHashTable('id', res.value);
+    const result = (await Cli.handleMultipleResultsFound(`Multiple Azure AD application registration with name '${name}' found. Choose the correct ID:`, `Multiple Azure AD application registration with name '${name}' found.`, resultAsKeyValuePair)) as { id: string };
```

Comment:
Much better, no? (in combination with my comment on cli.ts)

```suggestion
    const result = await Cli.handleMultipleResultsFound<{ id: string }>(`Multiple Azure AD application registration with name '${name}' found. Choose the correct ID:`, `Multiple Azure AD application registration with name '${name}' found.`, resultAsKeyValuePair);
```

Reply:
you mean to change the helper method to generic. Sure. Will refactor 👍